.eslintrc

```diff
@@ -41,6 +41,6 @@
     "no-use-before-define": 0,
     "no-unused-vars": [1, {"vars": "all", "args": "none"}],
     "no-underscore-dangle": 0,
-    "no-restricted-syntax": ["error", "ForInStatement", "LabeledStatement", "WithStatement"],
+    "no-restricted-syntax": ["error", "ForInStatement", "LabeledStatement", "WithStatement"]
```

Comment:
since you're not changing this rule, I would suggest reverting this single character change too.